tests/test_tasks/test_stat_task.py

```diff
@@ -0,0 +1,38 @@
+from time import time
+
+import openml
+from openml.testing import TestBase
+
+
+class OpenMLStatTest(TestBase):
```

Comment:
Could you please explain the naming here?

Reply:
Yes, good point.